lib/repl.js

```diff
@@ -1535,10 +1535,10 @@ function defineDefaultCommands(repl) {
     help: 'Save all evaluated commands in this REPL session to a file',
     action: function(file) {
       try {
-        fs.writeFileSync(file, this.lines.join('\n') + '\n');
-        this.outputStream.write('Session saved to: ' + file + '\n');
+        fs.writeFileSync(file, this.lines.join('\n'));
```

Comment:
The originally added line break caused the complete function not to work as expected. Saving should just plainly save exactly the users data and not add the extra line break, since that's not what the user actually did.